database/service_db.go

```diff
@@ -64,52 +87,119 @@ func (d *LevelDBServiceDB) All() ([]*service.Service, error) {
 				logrus.WithField("service", decodeErr.ID).Warning(decodeErr.Error())
 				continue
 			}
+			iter.Release()
```

Comment:
seems forgotten here (it shouldn't be called during iteration?)

Reply:
It's not forget.
It need to be done at the end of the iteration, the next line is a return, so it will stop the iteration

Reply:
that's right